### ISSUE TITLE
Stepper: JAWS reads aria-label for icons in non-active steps

### ISSUE THREAD

**Issue by nikhil4326** [angular/components#19009]
JAWS Screen reader is having issues while reading the content on 2nd step.

Steps to reproduce:
1. create stepper on page with three steps on it
2. On first step add field with mat-icon, icon should clickable with dynamic aria label

 
 
#### Expected Behavior
JAWS screen reader has option to read all the content of page. while reading content on 2nd stepper it should not read the content of 1st step.

#### Actual Behavior
JAWS screen reader is reading the mat-icon aria label of 1st stepper in 2nd stepper. This happen only in case when screen reader reads the total content of the screen(insert+v)

https://angular-material-7-sandbox-en8aoh.stackblitz.io/
##### Detail Explaination

We are using angular material stepper. On first page of stepper, JAWs is reading all the fields correctly. First page has a field with Icon . Field is having dynamic aria-label which can change on some condition. 
 
On read all (insert + down arrow), Its reading field and clickable icon of it along with icon Aria label. So no problem here.
Problem starts on 2nd page, when user filled the information on 1st step and moved to 2nd step(fill your address). When start read all(insert + down arrow). Its reading aria label of icon on first step(not whole page only icon).
Those icons(aria-label) are not present 2nd step even. This is happening only when aria-label is dynamic not when aria-label is static.
The same issue on 3rd step. Its reading dynamic aria-label of icon on first page and second page.

#### Environment

- Angular: 7.3.7
- CDK/Material:7.3.7
- Browser(s):
- Operating System (e.g. Windows, macOS, Ubuntu):

**Comment by jelbourn** [comment]
We don't have the ability to test with JAWS until we're back in the office, but I suspect this is either an issue with JAWS or the application code rather than the stepper/icon.

**Comment by nikhil4326** [comment]
@jelbourn  Are you able to on this issue?

**Comment by jelbourn** [comment]
Still not back in the office, so not yet

**Comment by nikhil4326** [comment]
@jelbourn When team will address this issue any rough idea?

**Comment by vinayshuklag** [comment]
Hi Team,
Any update on this issue, we are also facing similar issues with JAWS and mat Stepper

**Comment by nikhil4326** [comment]
@jelbourn Any workaround on this issue?

**Comment by jelbourn** [comment]
@amysorto was this something you ran into while working on stepper a11y recently?

**Comment by amysorto** [comment]
I didn't see this issue when working on stepper a11y. Since stepper icons should not be read aloud based on this PR: https://github.com/angular/components/pull/24025 this issue should be fixed. I also wasn't able to locally reproduce this issue.
